### DISCUSSION TITLE
How can I get via identify the image name from the IPTC profile?

### DISCUSSION THREAD

**Discussion by halloleo** [ImageMagick/ImageMagick#5024] [General]
I have JPG images which contain the image title in the IPTC metadata area under the key `Image Name` and I try to retrieve this with ImageMagick (version 6.9 and 7.1).

Via `identify -verbose` I can see that IM reads this fields as part of the IPTC profile, but I would like to print out only this value. So I though of an explicit percent escape along the lines of [ImageMagick – Format and Print Image Properties](https://imagemagick.org/script/escape.php). I tried:

    identify -format "%[IPTC:ImageName]"

but I get an "_unkown image property_" error back. This makes sense beacsue the Format and Print Image Properties page does not list this escape sequence.

Is there a generic way to get values from profiles via the format system?

**Comment by snibgo** [comment]
See my answer on your other thread.

If you supply a sample input, perhaps I can be more specific.

**Reply by halloleo** [reply]
Thanks for replying, @snibgo!

All sorted. See my other answer here.

**Comment by halloleo** [comment]
I didn't read the section about IPTC on the Format and Print Image Properties page properly. 

It is all there: In the format escape sequence I have to provide the _numbers_ which are shown in IPTC record list (bottom of the page) to the left of the key name. So for the Image Title (or in IPTC speak 'Object Name') I have to write the following command:

    identify -format "%[IPTC:2:05]"

and - voila! - I get the Image Title! 😄
